.storybook/addons/theme-addon/register.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+
+import {addons, types} from '@storybook/addons';
+import {useGlobals} from '@storybook/api';
+import type {API} from '@storybook/api';
+
+import {themes} from '../../theme';
+
+const ADDON_ID = 'yc-theme-addon';
+const TOOL_ID = `${ADDON_ID}tool`;
+
+addons.register(ADDON_ID, (api) => {
+    addons.add(TOOL_ID, {
+        type: types.TOOL,
+        title: 'Theme',
+        render: () => {
+            return <Tool api={api} />;
+        },
+    });
+});
+
+function Tool({api}: {api: API}) {
+    const [{theme}] = useGlobals();
+    React.useEffect(() => {
+        api.setOptions({theme: themes[theme]});
```

Comment:
the standard storybook theme switcher did not work?

Reply:
you can also change global theme via:
```
const [globals, updateGlobals] = useGlobals();
...
updateGlobals({theme});
...
```

Reply:
same, it is from uikit, just trying to keep gravity-ui projects similar